### PR TITLE
[TSD-149] Refactor project into submodules.

### PR DESCRIPTION
**Pull request by ksaric**
https://iohk.myjetbrains.com/youtrack/issue/TSD-149

Fixing the project structure so we can start adding the frontend.

### CODE REVIEW

File: web/src/Main.hs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module Main where
+
+import           Universum
+
+import           DataSource (TicketId (..), runApp)
+import           Lib (createBasicDataLayerIO, createConfig, exportZendeskDataToLocalDB, fetchAgents,
+                      fetchAndShowTickets, fetchAndShowTicketsFrom, fetchTickets,
+                      inspectLocalZipAttachment, processTicketSafe, processTickets,
+                      processTicketsFromTime, showStatistics)
```
Comment (HirotoShioi) [inline web/src/Main.hs]: I don't think these imports are needed

**Review by HirotoShioi — COMMENTED** [review]
When I run test on local machine with `stack test --fast`, the results are not showing on the console, but instead written in log file. Anyway I can see the result on the console?

File: web/src/Main.hs (hunk 1 above)
Comment (ksaric) [reply web/src/Main.hs]: Correct. Me and imports :grin:

**Comment by ksaric** [comment]
@HirotoShioi See - https://github.com/commercialhaskell/stack/blob/master/doc/build_command.md#build-output
You can use something like `stack test log-classifier:log-classifier-test` to build and test a single module that will show output.
Also, for now, `stack test --dump-logs` is an option.

**Review by HirotoShioi — APPROVED** [review]
Thank you for the changes. I'll approve it.

**Pull request merged**